go-kit/metrics/provider/librato/librato.go

```diff
@@ -223,6 +230,9 @@ type gauge struct {
 	Min    float64 `json:"min"`
 	Max    float64 `json:"max"`
 	SumSq  float64 `json:"sum_squares"`
+
+	// Attributes are used to enable SSA for gauges.
+	Attributes map[string]interface{} `json:"attributes,omitempty"`
```

Comment:
Librato's API docs have a [fixed list of attributes](https://www.librato.com/docs/api/#attributes). Maybe this should be a type instead of a type hole?